packages/components/src/dashicon/index.js

```diff
@@ -886,10 +887,18 @@ export default class Dashicon extends Component {
 		const iconClass = [ 'dashicon', 'dashicons-' + icon, className ].filter( Boolean ).join( ' ' );
 
 		return (
-			<SVG className={ iconClass }
-				size={ size }
-				path= { path }				
-			/>			
+			<SVG
+				aria-hidden
+				role="img"
+				focusable="false"
+				className={ iconClass }
+				xmlns="http://www.w3.org/2000/svg"
+				width={ size }
+				height={ size }
+				viewBox="0 0 20 20"
+			>
+				<Path d={ path } />
```

Comment:
I'm not sure this will work on the web. If you see my original code on my PR for the web the Path element is defined using lowercase. i.e: 'path'

Reply:
I see what you did know you redefined Path also.

Reply:
Hmm, I extracted the `Path` component to its own component and `Path` is imported here https://github.com/WordPress/gutenberg/pull/9064/files#diff-297ebddffcddddf088a365bab7f0909cR13. That should do it, although I think there's probably a problem with the way I've defined the web side variants (related to [the comment](https://github.com/WordPress/gutenberg/pull/9064#issuecomment-413734556)).

Reply:
From the comments I saw on the file I thought this was the case, if we could a pointer to where to make the changes it will be great.

Reply:
Dashicons are edited in https://github.com/WordPress/dashicons. Feel free to ping me for help, I can fasttrack any changes you'd like.

Reply:
I think this might be the file to modify on the Dashicons repo: https://github.com/WordPress/dashicons/blob/master/sources/react/index-footer.jsx. Not sure yet how to integrate that repo/project and have it hot-reloading in the React Native app but yeah, let's see.

Reply:
If you'd like to make a PR, I can approve it for you and ship it to Gutenberg.

Reply:
@youknowriad , it would still be good to review the proposal in this PR and agree on whether we like/want it or not before migrating the change to the Dashicons repo, so we take advantage of the quick turnaround time we enjoy with the Gutenberg+ReactNative repo combo.

Reply:
I wonder if the best approach here would be to have an alternative `index.native.js` for the whole dashicon component instead of extracting `Path` and `Svg`